sof_wrapper/audit.py

```diff
@@ -0,0 +1,23 @@
+"""Audit
+
+functions to simplify adding context and extra data to log messages destined for audit logs
+"""
+from flask import current_app
+from sof_wrapper.wrapped_session import get_session_value
+
+
+def audit_entry(message, level='info', extra=None):
+    """Log entry, adding in session info such as active user"""
+    try:
+        log_at_level = getattr(current_app.logger, level.lower())
+    except AttributeError:
+        raise ValueError(f"audit_entry given bogus level: {level}")
+
+    if extra is None:
+        extra = {}
+    extra['user'] = get_session_value('user')
```

Comment:
I think you need to check if user exists before adding it to `extra`?
```suggestion
    user = get_session_value('user')
    if user:
        extra['user'] = user
```

Reply:
sure, i know it always will, but code changes, good suggestion.